NodePropertyPalette/NodePropertyPaletteViewModel.cs

```diff
@@ -174,7 +175,19 @@ internal void ApplyBulkOperation()
 
         private void Disconnect(IEnumerable<PropertyPaletteNodeViewModel> selectedNodes)
         {
-            // TODO: Implement this
+            foreach (var node in selectedNodes)
+            {
+                // Materialize the enumeration first to modify it without causing errors
+                foreach (var connector in node.NodeModel.AllConnectors.ToList())
+                {
+                    // This would be the ideal. Unfortunately, the method is not publicly exposed.
+                    //connector.Delete();
```

Comment:
Thank you for finding this! Let's discuss tomorrow at standup with team if we can expose this API